cg/cli/workflow/rnafusion/base.py

```diff
@@ -102,52 +102,21 @@ def run(
 ) -> None:
     """Run rnafusion analysis for given CASE ID."""
     analysis_api: RnafusionAnalysisAPI = context.meta_apis[MetaApis.ANALYSIS_API]
-    analysis_api.status_db.verify_case_exists(case_internal_id=case_id)
-
-    command_args: CommandArgs = CommandArgs(
-        **{
-            "log": analysis_api.get_log_path(
-                case_id=case_id, workflow=analysis_api.workflow, log=log
-            ),
-            "work_dir": analysis_api.get_workdir_path(case_id=case_id, work_dir=work_dir),
-            "resume": not from_start,
-            "profile": analysis_api.get_profile(profile=profile),
-            "config": analysis_api.get_nextflow_config_path(
-                case_id=case_id, nextflow_config=config
-            ),
-            "params_file": analysis_api.get_params_file_path(
-                case_id=case_id, params_file=params_file
-            ),
-            "name": case_id,
-            "compute_env": compute_env or analysis_api.get_compute_env(case_id=case_id),
-            "revision": revision or analysis_api.revision,
-            "wait": "SUBMITTED",
-            "id": nf_tower_id,
-        }
+    analysis_api.run_nextflow_analysis(
+        case_id=case_id,
+        dry_run=dry_run,
+        log=log,
+        work_dir=work_dir,
+        from_start=from_start,
+        profile=profile,
+        config=config,
+        params_file=params_file,
+        revision=revision,
+        compute_env=compute_env,
+        use_nextflow=use_nextflow,
+        nf_tower_id=nf_tower_id,
```

Comment:
We need to also `try` and `except` here

Reply:
Also since this a different workflow we need to test the behaviour change.

Reply:
Behaviour tested, but will test again with try and except

Reply:
Moved run into nf-analysis CLI, see comment: https://github.com/Clinical-Genomics/cg/pull/2732#discussion_r1525930291